api/tests/utils/common_functions.py

```diff
@@ -79,6 +82,15 @@ def get_app_from_config_file(config_sample_path):
 # Get App from config file -- end
 
 
+# Get any section from config file -- begin
+def get_section_from_config_file(section, config_sample_path):
+    config_sample_json = get_config_file_json_strings(config_sample_path)
+    config_mapped_string = map_section_from_config(section, config_sample_json)
+    config_mapped = section_string_to_section_multi_type(config_mapped_string)
+    return config_mapped
+# Get any section from config file -- end
```

Comment:
why are these comments here?

Reply:
Personally, I did this to group functions that have a common purpose, but I can remove those comments if I want to.

Reply:
Yeah I think it adds a bit of clutter, If wanted you could add a single line docstring to each method, else the name should suffice.

Reply:
No problem, will be removed!